docs/users-guide/03-basic-exploration.md

```diff
@@ -69,6 +69,25 @@ Searches take into account items’ titles, descriptions, and other metadata —
 
 ![Search results](./images/basic-exploration/search-results.png)
 
+## Bookmarks
+
+**Bookmarks** are a way to quickly get back to things you visit frequently (or have been working on recently). Bookmarked items show up in the main navigation sidebar above [collections][collections].
```

Comment:
This may be obvious since bookmarks are presumably a familiar concept to most people using Metabase, but it may be worth specifying at the start that bookmarks are a thing that you control and create. The first sentence here makes them sound almost indistinguishable from a "recently/frequently viewed" list. (I don't feel particularly strongly about this, but worth considering)